data/wanderer/wanderers start.txt

```diff
@@ -1851,7 +1817,7 @@ mission "Wanderers Invaded 3C"
 	on offer
 		conversation
 			`You land on <origin>, and Emeka'a Isai leads you to the building where the council of war is taking place. Sayari, the Hai diplomat, has also been invited, along with Iktat Rek and various other leaders. Rek looks even more bedraggled than before: his wings are drooping, and more of his feathers are out of place.`
-			`	It is not entirely clear to you who outranks whom in the meeting, or even what roles the different participants have. To make matters worse, you are uncomfortably reminded that your translator only understands the Hai language. Rek and Isai speak mostly in Hai so that you and Sayari can be included, but when the conversation gets heated they all switch into their native tongue, a cacophony of clicks and whistles.`
+			`	It is not entirely clear to you who outranks whom in the meeting, or even what roles the different participants have. To make matters worse, they're talking much faster than your translator can process. Rek and Isai speak slowly so that you and Sayari can be included, but when the conversation gets heated they speed up, in a cacophony of clicks and whistles.`
```

Comment:
```suggestion
			`	It is not entirely clear to you who outranks whom in the meeting, or even what roles the different participants have. To make matters worse, they're talking much faster than your translator can process. Rek and Isai speak slowly so that you and Sayari can be included, but when the conversation gets heated they speed up in a cacophony of clicks and whistles.`
```